cnxeasybake/oven.py

```diff
@@ -254,6 +257,19 @@ def bake(self, element, last_step=None):
 
         # Do label/link updates
 
+    def report_coverage_zero(self, rule):
+        """Add entry to coverage saying this selector was parsed"""
+        self.coverage_lines.append('DA:{},0'.format(rule.source_line))
+
+    def report_coverage(self, rule):
+        """Add entry to coverage saying this selector was matched"""
+        logger.debug(u'Rule ({}): {}'.format(*rule))
+        self.coverage_lines.append('DA:{},1'.format(rule[0]))
```

Comment:
I was hoping to record each declaration too. Do you know of a good point in the code where I could get the `source_line` field for them?

Also, if there were declarations that easybake didn't understand (or didn't use because a string was empty or something) it would be a nice-to-have to not include them in the coverage

Finally, when there are 2 selectors (we have a policy of including each selector on a separate line) do you know how I could get the line number of  the selector that actually matched?

For example, in these lines only the 1st selector is marked as being covered:

![image](https://cloud.githubusercontent.com/assets/253202/20020969/48f5814c-a2aa-11e6-88f1-05a977b4809d.png)